api/server/server.go

```diff
@@ -898,6 +898,10 @@ func (s *Server) Datastore() models.Datastore {
 	return s.datastore
 }
 
+func (s *Server) Agent() agent.Agent {
+	return s.agent
+}
+
```

Comment:
note, that `Agent()` is not a part of `fnext.ExtServer` because it brings a lot of mess in code along with circular references:
```
import cycle not allowed
package github.com/fnproject/fn
	imports github.com/fnproject/fn/api/server
	imports github.com/fnproject/fn/api/agent
	imports github.com/fnproject/fn/fnext
	imports github.com/fnproject/fn/api/agent
```
and so on. I guess simple exposable method is enough.